tests/js-modules/modules.py

```diff
@@ -494,37 +551,23 @@ def test_npm_app(network, args):
         assert r.status_code == http.HTTPStatus.OK, r.status_code
         assert r.body.json() == False, r.body
 
-        key_priv_pem, key_pub_pem = infra.crypto.generate_ec_keypair("secp256r1")
-        algorithm = {"name": "ECDSA", "hash": "SHA-256"}
-        data = "foo".encode()
-        signature = infra.crypto.sign(algorithm, key_priv_pem, data)
-        r = c.post(
-            "/app/verifySignature",
-            {
-                "algorithm": algorithm,
-                "key": key_pub_pem,
-                "signature": b64encode(signature).decode(),
-                "data": b64encode(data).decode(),
-            },
-        )
-        assert r.status_code == http.HTTPStatus.OK, r.status_code
-        assert r.body.json() == True, r.body
-
-        key_priv_pem, key_pub_pem = infra.crypto.generate_ec_keypair("secp256k1")
-        algorithm = {"name": "ECDSA", "hash": "SHA-256"}
-        data = "foo".encode()
-        signature = infra.crypto.sign(algorithm, key_priv_pem, data)
-        r = c.post(
-            "/app/verifySignature",
-            {
-                "algorithm": algorithm,
-                "key": key_pub_pem,
-                "signature": b64encode(signature).decode(),
-                "data": b64encode(data).decode(),
-            },
-        )
-        assert r.status_code == http.HTTPStatus.OK, r.status_code
-        assert r.body.json() == True, r.body
+        curves = [ec.SECP256R1, ec.SECP256K1]
```

Comment:
With 384/512 we would use different hash algorithms, not worth adding to this test though I think.